### PR TITLE
Clear cache API - additional edits

### PR DESCRIPTION
**Pull request by alicejw-aws**
### Description
Editorial review comments for [Add the two clear cache APIs](https://github.com/opensearch-project/documentation-website/pull/1168) PR

### Issues Resolved
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by hdhalter — APPROVED** [review]
Looks good!

File: _opensearch/rest-api/index-apis/clear-index-cache.md [hunk 1]
```diff
@@ -8,34 +8,38 @@ nav_order: 320
 
 ## Clear index or data stream cache
 
-Clears the caches of one or more indexes. For data streams, the API clears the caches of the stream’s backing indexes.
+The cache clear API operation clears the caches of one or more indexes. For data streams, the API clears the caches of the stream’s backing indexes.
```
Comment (kolchfa-aws) [inline _opensearch/rest-api/index-apis/clear-index-cache.md:11]: "clear cache API"? That's how it's used below and I think it is more understandable in that order.

**Review by kolchfa-aws — APPROVED** [review]
LGTM. One small comment.

File: _opensearch/rest-api/index-apis/clear-index-cache.md (hunk 1 above)
Comment (alicejw-aws) [reply _opensearch/rest-api/index-apis/clear-index-cache.md]: thanks!

**Pull request merged**